bootstrap/postgres/configs.go

```diff
@@ -460,50 +460,58 @@ func (cr configRepository) DisconnectThing(ctx context.Context, channelID, thing
 	return nil
 }
 
-func (cr configRepository) retrieveAll(owner string, filter bootstrap.Filter) (string, []interface{}) {
-	template := `WHERE owner = $1 %s`
-	params := []interface{}{owner}
-	// One empty string so that strings Join works if only one filter is applied.
-	queries := []string{""}
-	// Since owner is the first param, start from 2.
-	counter := 2
+func (cr configRepository) retrieveAll(domainID string, thingIDs []string, filter bootstrap.Filter) (string, []interface{}) {
+	params := []interface{}{}
+	queries := []string{}
+
+	if domainID != "" {
+		params = append(params, domainID)
+		queries = append(queries, fmt.Sprintf("domain_id = $%d", len(params)))
```

Comment:
Why not?

```suggestion
		queries = append(queries, fmt.Sprintf("domain_id = %s", domainID))
```

Reply:
I think doing so would be risking SQL injection attacks